app/elements/elements.html

```diff
@@ -22,3 +22,5 @@
 
 <!-- TODO(victorbalan): investigate why this element cause the flickering -->
 <link rel="import" href="/elements/skill/skill-list.html">
+
+<link rel="import" href="/elements/app-element.html">
```

Comment:
why is this needed? You import `elements.html` [here](https://github.com/coduno/app/pull/112/files#diff-b89d9c11e31f0460fcd494ea9ec9764eR1) and in the `index.html` you import `app-element.html`. This looks like a useless import, and it also creates a circular dependency(which i don't know if it can cause any problems but better safe than sorry)